### DISCUSSION TITLE
http addon using Ingress, NGINX Ingress Controller and ExternalName Service

### DISCUSSION THREAD

**Discussion by shreya0910** [kedacore/http-add-on#992] [General]
I am trying to setup  keda + httpadd on is in keda namespace & application + ingress is in app ns. Created Ingress, NGINX Ingress Controller and ExternalName Service setup(in app ns) 

Here is the external service manifest
apiVersion: v1
kind: Service
metadata:
  name: demo-externalname
  namespace: app
spec:
  type: ExternalName
  externalName: keda-add-ons-http-interceptor-proxy.keda.svc.cluster.local 

 but so far its not working as expected
keda-add-ons-http-interceptor-proxy is not reachable from nginx. Here is the log

[26/Apr/2024:22:31:42 +0000] "GET /path HTTP/1.1" 499 0 "-" "-" 120 15.993 [apps-keda-add-ons-http-interceptor-proxy-8080] []            
[26/Apr/2024:22:31:42 +0000] "GET /path  HTTP/1.1" 502 150 "-" "-" 120 2.025 [apps-keda-add-ons-http-interceptor-proxy-8080] [] 

Has anyone setup in similar way and achieved success

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
